### ISSUE TITLE
Stickers for UofT Coders

### ISSUE THREAD

**Issue by linamnt** [UofTCoders/council#69]
Apparently they can be fairly cheap. Will investigate!

**Comment by linamnt** [comment]
Details:
- 2"x2" circle stickers with our logo (I think that's a reasonable size, but do we want to go bigger?)
- Vinyl (like the Mozilla Science Fox ones, not paper as vinyl makes them more laptop friendly)

Cheapest deal I've found is https://www.stickermule.com/ for $80 (free shipping) for 200 stickers

**Comment by mbonsma** [comment]
I think 2" diameter is great! I'm super excited for this!

**Comment by lcoome** [comment]
two sounds perfect [😊]


yay stickers

________________________________

**Comment by linamnt** [comment]
Cool! If there are no objections, I can place an order soon and we should get them within 5 business days :D

**Comment by mbonsma** [comment]
@lwjohnst86, any objections from the treasurer to purchasing stickers?

**Comment by lwjohnst86** [comment]
Nope! I think it's a valuable expense! :smile:

**Comment by linamnt** [comment]
So I commissioned my lab mate (and by commissioned I mean bribed with cookies) to clean up the logo a bit before we printed them onto stickers. He actually went an extra step and made a new logo:

![uoftcodersgrey](https://cloud.githubusercontent.com/assets/10761918/22892249/e171c8d4-f1df-11e6-8cdf-c4429e096fec.png)
![uoftcodersyellow](https://cloud.githubusercontent.com/assets/10761918/22892248/e17084ce-f1df-11e6-89bb-067bb16877a7.png)

It's very different, but I personally really like it for the simplicity. The yellow is meant to match the website, and the grey for more general use. However, I understand sticking to what we have for consistency, or more of the coding aspect of it? Thoughts? 

He can modify it of course if we have requests, or we can stick with the original logo for the stickers and going forward.

**Comment by lcoome** [comment]
I vote yellow!!!!

________________________________

**Comment by lcoome** [comment]
Like, all the time. Even for general use. The yellow really works.

________________________________

**Comment by joelostblom** [comment]
Sweet! I also like the yellow!

**Comment by mbonsma** [comment]
Whoah cool! I think I like both the yellow and the old one equally. What if the text was a bit bigger, so that it filled more of the circle? I'm not a design person though so if you're happy with it, I agree!

**Comment by linamnt** [comment]
yeah agreed, I asked him that too re: filling the circle more! Maybe if we hear from @lwjohnst86 and @QuLogic about their thoughts too, and then we can go forward with ordering the finalized sticker we decide upon.

**Comment by QuLogic** [comment]
They look pretty good. For interest's sake, what does the yellow one look like inverted? Or maybe the grey one with the lightbulb (maybe the CN tower, too) in yellow? Not sure if they'd look any better...

**Comment by linamnt** [comment]
@QuLogic here they are below. I personally like the yellow + yellow inverted (first and last)?

![uoftcoderslargeyellow](https://cloud.githubusercontent.com/assets/10761918/23007664/f121bd76-f3d7-11e6-839a-86db027656b6.png)
![uoftcodersyellowiconsv1](https://cloud.githubusercontent.com/assets/10761918/23007665/f121f1e2-f3d7-11e6-9532-9d478befe84a.png)
![yelowiconsversion](https://cloud.githubusercontent.com/assets/10761918/23007666/f1229214-f3d7-11e6-9ff2-c4c642b33210.png)
![darkbackground](https://cloud.githubusercontent.com/assets/10761918/23007667/f125b07a-f3d7-11e6-911f-05b47e7ba777.png)

**Comment by lcoome** [comment]
My vote is first and third, with first being a strong win for me.

________________________________

**Comment by mbonsma** [comment]
I'm with @linamnt, I like the first and last.

**Comment by joelostblom** [comment]
1. Last
2. First

**Comment by linamnt** [comment]
I think the stickers as well as general purpose logo should be the yellow one, and the grey ones would be website or for other uses when in need of a darker theme?

**Comment by QuLogic** [comment]
If we're updating the website too, let's see how that looks. Note that these images are not totally correct as the logo is _way_ too big at the moment, and it's scaled down quite poorly by browsers.

Black on yellow:
![newlogo1-top](https://cloud.githubusercontent.com/assets/302469/23054332/546dda88-f4ad-11e6-9be6-6af2fb0d2c87.png)
![newlogo1-down](https://cloud.githubusercontent.com/assets/302469/23054331/5227ce46-f4ad-11e6-8e7d-149d396d2925.png)

Black on grey with yellow highlight:
![newlogo2-top](https://cloud.githubusercontent.com/assets/302469/23054349/65bdaec6-f4ad-11e6-9d4b-e7245a93b99c.png)
![newlogo2-down](https://cloud.githubusercontent.com/assets/302469/23054350/670d683e-f4ad-11e6-9af2-225b0555bd64.png)

Black on yellow with a little less yellow highlight:
![newlogo3-top](https://cloud.githubusercontent.com/assets/302469/23054362/73bd830c-f4ad-11e6-9715-d70fe8525bd9.png)
![newlogo3-down](https://cloud.githubusercontent.com/assets/302469/23054363/7505916e-f4ad-11e6-8b77-8480ed779e2f.png)

Yellow on grey:
![newlogo4-top](https://cloud.githubusercontent.com/assets/302469/23054375/8063a1f4-f4ad-11e6-9cb0-812c62527654.png)
![newlogo4-down](https://cloud.githubusercontent.com/assets/302469/23054377/81e774a6-f4ad-11e6-8ac6-26762a3197c1.png)

Do not worry about the "crispness"; once we pick one, it needs to be tweaked to work nicely at the lower resolution.

**Comment by lcoome** [comment]
dark grey or yellow work for the website, definitely not light grey

________________________________

**Comment by lwjohnst86** [comment]
Woooowwww, those look great! (Sorry for the late reply). I definitely vote 1 and 4 as they both work well for the website! Wow, I really like those... love the CN tower there!

**Comment by linamnt** [comment]
We can order both the 1st and 4th, maybe 100 each? or just the yellow (which I think everyone at least agrees is part of their top 2) for 200 of these? It's $80 (free shipping) for 200 stickers. I think that's a good start.

**Comment by QuLogic** [comment]
Where do you get $80? Is that USD?

Also, this is pretty minor, but in the enlarged version, the C and s are not quite the same distance from the edge of the circle. This is not evident at website size, but it might be noticeable on the stickers.

**Comment by linamnt** [comment]
https://www.stickermule.com/products/circle-stickers
I choose 2x2" and 200, and yes I forgot to check but that is USD, in CAD it is $100

**Comment by linamnt** [comment]
here it is with Coders shifted. 
![uoftcoderslargeyellowaligned](https://cloud.githubusercontent.com/assets/10761918/23184745/f9a62a7e-f84e-11e6-93c8-a8e2331b36b5.png)

**Comment by linamnt** [comment]
Arrived yesterday, will bring at the next meeting (and tomorrow when we meet)

**Comment by mbonsma** [comment]
We're starting to get low on stickers - should we order another round?

**Comment by linamnt** [comment]
100?

**Comment by QuLogic** [comment]
Make sure to use the correctly aligned one! 😉

**Comment by mbonsma** [comment]
@linamnt yeah 100 of each sounds good!

**Comment by joelostblom** [comment]
Hey guys, would we be interested in buying/requesting other stickers promoting openness (linux, ubuntu, python, R, mozilla, bioarchive, CC, etc) and handing out as well? Or shall we not use our budget on such extravagance?

**Comment by linamnt** [comment]
Just got 200 new stickers, thanks @QuLogic for fixing the grey sticker! 
They'll be here in two weeks :confetti_ball: :confetti_ball: 

@joelostblom I think that'd be cool, should they be from the companies themselves and do they all have different places they sell them? Otherwise http://www.unixstickers.com/stickers seems to have most of the ones you listed.

**Comment by joelostblom** [comment]
Yes unixstickers is where I saw many of them also. I think getting them
from there is fine, it's  cheaper in bulk, so maybe we can wrangle some
academic additional bulk discount...

**Comment by mbonsma** [comment]
I think that's a fine idea although I would love to get them for free if possible!
